README.md

```diff
@@ -1 +1,9 @@
-# Earth System Metrics and Diagnostics Standards
+# Earth System Metrics and Diagnostics Standards (EMDS)
```

Comment:
Looks good!  This is great to get things started.

Eventually this front page should include a statement about "why" (i.e. the mission of the effort) and "who" (i.e. the membership and governance).